components/cards/CardButton.jsx

```diff
@@ -33,13 +33,21 @@ const CardButton = ({
         text: "card__button--borderless"
     };
 
+    const handleKeyUp = (e) => {
+        if (e.key === "Enter") {
+            action();
+        }
+    };
+
     if (onlyPerformAction === true) {
         return (
             <div className="card__button">
                 <button
                     className={`card__button--secondary ${variantMapper[variant]} ${customClassName}`}
+                    tabIndex="0"
```

Comment:
Just wondering why the tab index is necessary here.  It shouldn't be necessary on a button element.  Also I can tab to this button currently in PROD without this addition.

Reply:
this one can be removed, i'll do that